isis/appdata/serialnumbers/CsmSerialNumber.trn

```diff
@@ -1,6 +1,6 @@
 Group = Keyword1
-  Auto
-  InputKey       = CSMPlatformId
+  Auto           = 0|1
```

Comment:
This should be Auto = 1 for true. This should also propagate to all `.trn` files in ISIS